ui/i18n/messages.json

```diff
@@ -16,14 +16,19 @@
   "command.cancel": "Cancel",
   "command.change.password": "Change Password",
   "command.clear": "Clear",
+  "command.close": "Close",
```

Comment:
Once this is merged, please don't forget to notify @manonlumeau.

Reply:
will do it